src/main/java/net/mcreator/element/types/Block.java

```diff
@@ -143,6 +144,12 @@
 	public Procedure bonemealSuccessCondition;
 	public Procedure onBonemealSuccess;
 
+	public boolean sensitiveToVibration;
+	public List<GameEventEntry> vibrationalEvents;

```

Comment:
This list needs to be initiated to an empty array list for backwards compatibility. This is what likely fails the tests